<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchStudentDialog</class>
 <widget class="QDialog" name="SearchStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Поиск студента</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="gLayout_1">
     <item row="0" column="0">
      <widget class="QLabel" name="info_lbl_2">
       <property name="text">
        <string>Имя ученика</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="student_name_le"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="info_lbl_1">
       <property name="text">
        <string>Тип валюты</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="currency_box">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="info_lbl_3">
     <property name="text">
      <string>Найденные ученики</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="students_list"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
